src/module/entities/helper.js

```diff
@@ -6,97 +6,590 @@
  */
 
 // Dependencies
-
+const entityTypesHelper = require(MODULES_BASE_PATH + '/entityTypes/helper')
+// const elasticSearch = require(ROOT_PATH + "/generics/helpers/elasticSearch");
+// const userRolesHelper = require(MODULES_BASE_PATH + "/userRoles/helper");
+const FileStream = require('../../generics/file-stream')
+// const userProfileService = require("../../generics/");
 
 const { v4: uuidv4 } = require('uuid')
 
 // const entitiesHelper = require(MODULES_BASE_PATH + "/entities/helper")
-const _ = require("lodash")
+const _ = require('lodash')
 // const programUsersQueries = require(DB_QUERY_BASE_PATH + "/programUsers")
 
 /**
-    * UserProjectsHelper
-    * @class
-*/
+ * UserProjectsHelper
+ * @class
+ */
 
 module.exports = class UserProjectsHelper {
+	static entityDocuments(findQuery = 'all', fields = 'all', limitingValue = '', skippingValue = '', sortedData = '') {
```

Comment:
@MallanagoudaB create a folder databaseQueries and create entities.js and add all the database calls there. Refer elevate unnati for reference